app/assets/javascripts/app/form-validation.js

```diff
@@ -1,16 +1,84 @@
 import 'classlist.js';
-import gf from 'gentleform';
+import H5F from 'h5f';
 
 
-function validateForm() {
-  const form = document.querySelector('form[novalidate]');
+// standardize validation messages across browsers
+function validateMessages(field) {
+  if (field.validity.valueMissing) {
+    return field.setCustomValidity('Please fill in all required fields.');
+  } else if (field.validity.typeMismatch) {
+    return field.setCustomValidity('Please match the requested format.');
```

Comment:
doesn't feel worth it to me for these minor and short strings (and may mess up the transpiling / minifying)